src/rebar3_hex_owner.erl

```diff
@@ -179,6 +178,12 @@ list(HexConfig, Package, State) ->
             ?PRV_ERROR({error, Package, Reason})
     end.
 
+owner_email(Owner) ->
+    case maps:get(<<"email">>, Owner, <<"">>) of
+        Email when is_binary(Email) -> binary_to_list(Email);
+        _ -> "unspecified"
```

Comment:
Technically, we can opt to print the username here vs the email, what do you think?

Reply:
Does that field always exist? If so, we could output `Name (Email)` always.

Reply:
I like that idea! One I like even better is printing it as a table, but I think that's another PR.